helper/linux/writejob.cpp

```diff
@@ -28,9 +28,11 @@
 #include <QDBusInterface>
 #include <QDBusUnixFileDescriptor>
 
-#include <sys/mman.h>
 #include <unistd.h>
-#include <stdlib.h>
+
+#include <cstddef>
+
+#include <tuple>
```

Comment:
I'm not sure about header ordering at all. Would love to know where to put what headers and in what order.